src/main/java/Duke.java

```diff
@@ -1,10 +1,82 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static void printTasks(Task[] tasks, int taskCount) {
+        System.out.println("Here are the tasks in your list:");
+        int taskNumber = 1;
+        for (int i = 0; i < taskCount; i++) {
+            System.out.print(taskNumber + ".");
+            tasks[i].printTask();
+            taskNumber++;
+        }
+    }
+
+    public static void markTask(Task[] tasks, int taskNumber) {
+        System.out.println("Nice! I've marked this task as done:");
+        tasks[taskNumber].markAsDone();
+        System.out.print(" ");
+        tasks[taskNumber].printTask();
+    }
+
+    public static void unmarkTask(Task[] tasks, int taskNumber) {
+        System.out.println("OK, I've marked this task as not done yet:");
+        tasks[taskNumber].markAsNotDone();
+        System.out.print(" ");
+        tasks[taskNumber].printTask();
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Ms Chatty :)");
+        System.out.println("What can I do for you?");
+
+        String userCommand;
+        Task[] tasks = new Task[100];
+        String[] arrayOfCommand = new String[4];
+        int taskCount = 0;
+        Scanner in = new Scanner(System.in);
+        userCommand = in.nextLine();
+
+        while (!(userCommand.equals("bye"))) {
```

Comment:
can consider moving while loop to a separate function to declutter main function